src/scrappers/source/proCyclingStats/races.js

```
@@ -128,6 +128,46 @@ export function extractRawRaceData(tableRows, year) {
     .filter((record) => record !== null);
 }
 
+/**
+ * Extracts raw race data from table row elements
+ * @param {Array<Element>} tableRows - Array of table row elements
+ * @param {number} year - The year of the races
+ * @returns {Array<RaceRecord>} Array of raw race records
+ */
+export function extractRawRaceData(tableRows, year) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'extractRawRaceData'. Consider to delete it or rename it.

'extractRawRaceData' is defined here:


(lint/suspicious/noRedeclare)